Retrospective/counter.js

```diff
@@ -0,0 +1,23 @@
+function createCounter(initialValue) {
```

Comment:
хорошо, но лучше `value` хранить в замыкании, чем в объекте, чтобы нельзя было напрямую изменить
+ удобнее наверное использовать дефолтные значения параметров 

```suggestion
function createCounter(initialValue = 0) {
  let value = initialValue;

  return {
    showValue() { return value; },
    increment(val = 1) {
      value += val;
      return this;
    },
    decrement(val = 1) {
      value -= val;
      return this;
    },
    discard() {
      value = initialValue;
      return this;
    },
  };
}
```